CHANGELOG.md

```diff
@@ -3,6 +3,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.0] - TBD
+### Added
+- SQS permissions policy.
+- tags to the lambda.
+
+### Changed
+- converting the filter_policy to a template.
+- updating filter vars from string to list.
```

Comment:
Similar to comment below, I think this is a backwards incompatible change? Which is fine,we just need to update the version number accordingly.